libs/shared/design-system/interfaces/BUILD.bazel

```diff
@@ -3,7 +3,7 @@ load("@npm_bazel_typescript//:index.bzl", "ts_config", "ts_library")
 package(default_visibility = ["//visibility:public"])
 
 filegroup(
-    name = "barista-definitions",
+    name = "design-system-interfaces",
```

Comment:
Thx. for taking care of the BAZEL files as well but please update the `/apps/barista-design-system/BUILD.bazel` file as well so that `yarn bazel build //apps/barista-design-system` is passing.

Reply:
Changed the reference in the barista-design-system bazel file.